sample/src/main/java/com/cloudinary/android/sample/app/ImageActivity.java

```diff
@@ -102,14 +97,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
 
     private void initExoPlayer() {
-        BandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
-        TrackSelection.Factory videoTrackSelectionFactory =
-                new AdaptiveTrackSelection.Factory(bandwidthMeter);
-        TrackSelector trackSelector =
-                new DefaultTrackSelector(videoTrackSelectionFactory);
-
-        exoPlayer = ExoPlayerFactory.newSimpleInstance(this, trackSelector);
-        exoPlayerView = ((SimpleExoPlayerView) findViewById(R.id.exoPlayer));
+//        DefaultBandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
```

Comment:
It's not clear what is the purpose of the commented out lines code.

If not needed, please remove

Reply:
Deleted, thank you